docs/index.md

```diff
@@ -94,7 +94,7 @@ you can install via pip, conda-forge, or from source.
 
 ### From pip, with "batteries included"
 
-napari can be installed on most macOS, Linux, and Windows systems with Python
+napari can be installed on most macOS (Intel x86), Linux, and Windows systems with Python
```

Comment:
May we put somewhere information about rosetta? (maybe a short note with a link somewhere on the apple website). Totally for newbies.

Reply:
I can try to write something up but making a rosetta2 env reliably is a bit tricky—you almost have to mess around with your conda install and Terminal.app. And then using it is also somewhat tricky. For me, it's honestly a bit too complex. It's much simpler to just run arm64 native. Apple docs describe the limitations of rosetta2, rather than how to use.

Reply:
In contradiction to me, you have experience. Maybe your text should focus on information that this is not the best idea. But will inform user about options that he may know exist (without details).

Reply:
I'm 👎 on including Rosetta discussion. Docs have to strike a balance between thoroughness and simplicity, and including Rosetta pushes it too far away from simplicity imho.

I also think that this text is already good, and rather than hold up this PR, we should make an issue where we can discuss whether we want a Rosetta discussion also.

Reply:
I'm also 👎 on including a Rosetta discussion - although the instructions are different, there are ways to get `napari` working natively on macOS arm, so I don't think discussing Rosetta is neccesary.

Reply:
I'm with @dstansby there seems no reason to use Rosetta2. And if we figure out and provide instructions, then we will need to offer at least some support, right? 👀

Reply:
I'm with @dstansby there seems no reason to use Rosetta2. And if we figure out and provide instructions, then we will need to offer at least some support, right? 👀